app/src/main/cpp/skyline/gpu/syncpoint.h

```diff
@@ -7,7 +7,7 @@
 
 namespace skyline {
     namespace constant {
-        constexpr size_t MaxHwSyncpointCount = 192; //!< The maximum number of HOST1X syncpoints on t210
+        constexpr size_t MaxHwSyncpointCount{192}; //!< The maximum number of HOST1X syncpoints on t210
```

Comment:
'Host1x'

Reply:
The context is `syncpoint.h` and there isn't really any other "Syncpoints" that this would be conflated with that we need to explicitly specify it in the name, we can stay less verbose.

Reply:
No no you misunderstood